{
  "vulinfoid": "JVNDB-2019-014058",
  "vulinfodata": {
    "title": "python-ecdsa におけるデジタル署名の検証に関する脆弱性",
    "vulinfodescription": {
      "overview": "python-ecdsa には、デジタル署名の検証に関する脆弱性が存在します。 "
    },
    "affected": {
      "affecteditem": [
        {
          "name": "python-ecdsa project",
          "productname": "python-ecdsa",
          "cpe": {
            "text": "cpe:/a:python-ecdsa_project:python-ecdsa",
            "version": "2.2"
          },
          "versionnumber": [
            "0.13.3 未満"
          ]
        },
        {
          "name": "レッドハット",
          "productname": "Red Hat Ceph Storage",
          "cpe": {
            "text": "cpe:/a:redhat:ceph_storage",
            "version": "2.2"
          }
        },
        {
          "name": "レッドハット",
          "productname": "Red Hat Enterprise Virtualization",
          "cpe": {
            "text": "cpe:/a:redhat:enterprise_virtualization",
            "version": "2.2"
          }
        },
        {
          "name": "レッドハット",
          "productname": "Red Hat OpenStack",
          "cpe": {
            "text": "cpe:/a:redhat:openstack",
            "version": "2.2"
          }
        }
      ]
    },
    "impact": {
      "cvss": [
        {
          "version": "2.0",
          "severity": {
            "text": "Medium",
            "type": "Base"
          },
          "base": "6.4",
          "vector": "AV:N/AC:L/Au:N/C:P/I:P/A:N"
        },
        {
          "version": "3.0",
          "severity": {
            "text": "Critical",
            "type": "Base"
          },
          "base": "9.1",
          "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:N"
        }
      ],
      "impactitem": {
        "description": "情報を取得される、および情報を改ざんされる可能性があります。"
      }
    },
    "solution": {
      "solutionitem": {
        "description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
      }
    },
    "related": {
      "relateditem": [
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "ecdsa 0.13.3",
          "url": "https://github.com/warner/python-ecdsa/releases/tag/python-ecdsa-0.13.3"
        },
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "Inconsistent handling of malformed DER signatures #114",
          "url": "https://github.com/warner/python-ecdsa/issues/114"
        },
        {
          "type": "vendor",
          "name": "PyPI",
          "vulinfoid": "ecdsa 0.13.3",
          "url": "https://pypi.org/project/ecdsa/0.13.3/"
        },
        {
          "type": "vendor",
          "name": "Red Hat Bugzilla",
          "vulinfoid": "Bug 1760843",
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14859"
        },
        {
          "type": "advisory",
          "name": "Common Vulnerabilities and Exposures (CVE)",
          "vulinfoid": "CVE-2019-14859",
          "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-14859"
        },
        {
          "type": "advisory",
          "name": "National Vulnerability Database (NVD)",
          "vulinfoid": "CVE-2019-14859",
          "url": "https://nvd.nist.gov/vuln/detail/CVE-2019-14859"
        },
        {
          "type": "cwe",
          "name": "JVNDB",
          "vulinfoid": "CWE-347",
          "url": "https://cwe.mitre.org/data/definitions/347.html",
          "title": "デジタル署名の不適切な検証"
        }
      ]
    },
    "history": {
      "historyitem": [
        {
          "historyno": "1",
          "datetime": "2020-01-27T17:49:49+09:00",
          "description": "[2020年01月27日]\\n  掲載"
        }
      ]
    },
    "datefirstpublished": "2020-01-27T17:49:49+09:00",
    "datelastupdated": "2020-01-27T17:49:49+09:00",
    "datepublic": "2019-10-07T00:00:00+09:00"
  }
}
